cli/flags.rs

```diff
@@ -360,6 +360,7 @@ fn types_parse(flags: &mut Flags, _matches: &clap::ArgMatches) {
 }
 
 fn fmt_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
+  flags.watch = matches.is_present("watch");
```

Comment:
Would be great to have unit test added for respective commands

Reply:
I forgot to do that, thanks! I've added tests for `bundle --watch` and `fmt --watch`.